tap4j/src/test/java/org/tap4j/consumer/issue3504508/TestIssue3504508.java

```diff
@@ -55,16 +55,21 @@ public void testTapConsumer() {
 
     @Test
     public void testProducingSubtests() {
-        final String expected = "1..3\n" + "ok 1 - First test\n" + "    1..2\n"
-                + "    ok 1 - This is a subtest\n" + "    ok 2 - So is this\n"
-                + "        1..2\n" + "        ok 1 - This is a subtest\n"
-                + "        ok 2 - So is this\n" + "ok 2 - An example subtest\n"
+        final String expected = "1..3\n"
+                + "ok 1 - First test\n"
+                + "    1..2\n"
+                + "    ok 1 - This is a subtest\n"
+                + "    ok 2 - So is this\n"
+                + "        1..2\n" 
+                + "        ok 1 - This is a subtest\n"
+                + "        ok 2 - So is this\n" 
+                + "ok 2 - An example subtest\n"
                 + "ok 3 - Third test\n";
```

Comment:
Oups, probably CTRL +SHIFT + F in Eclipse for auto formatting. Sorry and thanks for fixing that.